transport_service_queries/src/main/java/pl/edu/pg/transport/Transport.java

```diff
@@ -0,0 +1,39 @@
+package pl.edu.pg.transport;
+
+import lombok.Getter;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.time.LocalDateTime;
+
+@Getter
+@Document("transports")
+public class Transport {
+
+    @Id
+    private Long id;
+
+    private LocalDateTime departureDate;
+    private String departureCountry;
+    private String departureCity;
+    private LocalDateTime arrivalDate;
+    private String arrivalCountry;
+    private String arrivalCity;
+    private String meanOfTransport;
+    private int seats;
+
+    @Override
+    public String toString() {
```

Comment:
Since you are using lombok, why not generate the `toString` in the same way as getters?

Reply:
Actually I don't need `toString()` ;)